### ISSUE TITLE
Post preview works incorrectly

### ISSUE THREAD

**Issue by shelkmike** [ialbert/biostar-central#937]
Hi. First, thank you for making and maintaining Biostars.

I noticed that sometimes post preview works incorrectly.
For example, when I want to add an empty line. As far as I know, there are two ways to add empty lines in Markdown.

1. The first is to use the \<br> tag. The will be an empty line in the post. But the preview will show just the text "\<br>".
2. The second is to use "\\", as suggested here https://stackoverflow.com/a/64164898 . This works vice-versa: there will be an empty line in the preview, but the post will have a "\\" character instead of an empty line.

**Comment by ialbert** [comment]
Yes, unfortunately, this is the effect of Markdown not having a strict definition (at least at the time we implemented the features), for the preview we are using a Javascript library, but after submission we are using a Python library to process the Markdown. The two implementations differ.
